sympy/series/tests/test_nseries.py

```diff
@@ -444,6 +445,21 @@ def test_floor():
     assert floor(x + 1.5).series(x) == 1
 
 
+def test_frac():
+    assert frac(x).series(x, 0, 6, cdir=1) == x + O(x**6)
+    assert frac(x).series(x, 0, 6, cdir=-1) == 1 + x + O(x**6)
+    assert frac(2*x).series(x, 0, 6, cdir=-1) == 1 + 2*x + O(x**6)
+    assert frac(2*x + 1).series(x, 0, 6, cdir=1) == 2*x + O(x**6)
+    assert frac(2*x + 2).series(x, 0, 6, cdir=1) == 2*x + O(x**6)
+    assert frac(x**2).series(x, 0, 6, cdir=1) == x**2 + O(x**6)
+    assert frac(x**2).series(x, 0, 6, cdir=1) == x**2 + O(x**6)
+    assert frac(x**3 - 1).series(x, 0, 6, cdir=-1) == 1 + x**3 + O(x**6)
+    assert frac(x**3 - 1).series(x, 0, 6, cdir=1) == x**3 + O(x**6)
+    assert frac(x**8).series(x, 0, 6, cdir=1) == O(x**6)
+    assert frac(1/x).series(x, 0, 6) == AccumBounds(0, 1) + O(x**6)
+    assert frac(1/x).series(x, 0, 0).expr == S.One
```

Comment:
What is the point of this test?

Reply:
Not sure if we want that , we would end up with O(1).expr for which I've returned 1,but its not too necessary a test !